src/pages/quiz-page.tsx

```diff
@@ -0,0 +1,196 @@
+import * as React from 'react';
+import { ReactElement, useState } from 'react';
+import { toast } from 'react-toastify';
+import {
+  Button, ButtonProps, Container, Form, Header, Radio,
+} from 'semantic-ui-react';
+import {
+  MESSAGE_ANSWER_CORRECT,
+  MESSAGE_ANSWER_WRONG,
+  BUTTON_END_QUIZ_TEXT,
+  BUTTON_NEXT_TEXT,
+  BUTTON_SKIP_TEXT,
+  BUTTON_SUBMIT_TEXT,
+  TOAST_SKIP_QUESTION,
+  LEVEL,
+  SCORE,
+  toastConfig,
+  BUTTON_DISABLE_CLASS_NAME,
+  BUTTON_ENABLE_CLASS_NAME,
+  RADIO_CORRECT_COLOR,
+  RADIO_WRONG_COLOR,
+  QUESTION,
+  BUTTON_END_QUIZ_COLOR,
+  FOOTER_MARGIN,
+  BUTTON_WIDTH_DEFAULT,
+  FONT_SIZE_MEDIUM,
+} from '../contants';
+
+const styles = {
+  containerFlex: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    flexDirection: 'column',
+    height: '80vh',
+  },
+  containerHeader: {
+    flex: '1',
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    height: '100%',
+    fontSize: FONT_SIZE_MEDIUM,
+    fontWeight: 'bold',
+  },
+  containerQuiz: {
+    flex: '3',
+    display: 'flex',
+    justifyContent: 'center',
+    flexDirection: 'column',
+  },
+  containerFooter: {
+    flex: '1',
+    display: 'flex',
+    justifyContent: 'center',
+    flexDirection: 'column',
+  },
+  containerButtons: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    height: '100%',
+  },
+  radio: {
+    textAlign: 'left',
+    width: '100%',
+  },
+  buttonWrapper: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  button: {
+    width: BUTTON_WIDTH_DEFAULT,
+  },
+};
```

Comment:
Do you want to separate this into its own CSS file?

Reply:
Yes this is a great idea. The `Button` components are imported from `semantic-ui`. Maybe there are better ways to override `semantic-ui`'s button width in theme config. I will do some research later.